README.md

```diff
@@ -211,4 +212,4 @@ A: Theoretically, all yes.
 
 ## Installation help
 
-Go read the original SM64 repo README.md
+Go read the original SM64 repo README.md
```

Comment:
Wait what even changed here? But tbh no please don't; the ultrasm64 stuff should probably be purged eventually but people should really be following the HackerSM64 wiki instead.

Reply:
oh I'm honestly not sure. This readme is nearly identical to the one for 2.2 minus the lines I added above.

Reply:
This random diff kinda disappeared somehow so this is resolved.